src/emoji/__tests__/emojiSelectors-test.js

```diff
@@ -17,30 +17,35 @@ describe('getActiveRealmEmoji', () => {
       ],
       realm: {
         emoji: {
-          smile: {
```

Comment:
Ideally, you would merge this commit with the change that made the tests failing.

Reply:
These tests were never failing, it's just that the structure of the objects used in these tests where not the same as the actual objects received from the server. That didn't cause the tests to fail, but I thought it's better to have accurate object strictures in these tests.

The new test I added will fail if the structure is wrong.